### ISSUE TITLE
`ftl kotlin init` generates a pom.xml with a hard coded local endpoint

### ISSUE THREAD

**Issue by alec-brooks** [TBD54566975/ftl#622]
At the moment, when generating a kotlin project, the pom.xml contains an invocation of the `ftl-generator` command. By default it gets generated with an endpoint that is hardcoded to `http://127.0.0.1:8892`. This makes deployment to other FTL clusters tricky, as it forces you to change this value in order to compile for different environments.

It would be ideal if this value could somehow be set dynamically. Regular FTL commands use the `FTL_ENDPOINT` environment variable. Perhaps this could be either read in the `ftl-generate` command or used in the pom.xml file to set this endpoint command line option.

This may be relevant to work being done by @worstell on the Kotlin front.

**Comment by alecthomas** [comment]
Yeah I think we should remove the flag from the pom and make `ftl-generator` use `${FTL_ENDPOINT}` if provided and if not, default to `http://127.0.0.1:8892`.
